benchexec/tools/cbmc.py

```diff
@@ -97,17 +103,18 @@ def isErrorMessage(msg):
             else:
                 status = "INVALID OUTPUT"
                 logging.exception(
-                    "Error parsing CBMC output for returncode %d", returncode
+                    "Error parsing CBMC output for exit_code.value %d", exit_code.value
                 )
 
         return status
 
-    def determine_result(self, returncode, returnsignal, output, isTimeout):
+    def determine_result(self, run):
+        output = run.output
 
-        if returnsignal == 0 and ((returncode == 0) or (returncode == 10)):
+        if run.exit_code.value in [0, 10]:
             status = result.RESULT_ERROR
             if "--xml-ui" in self.options:
-                status = self.parse_XML(output, returncode, isTimeout)
+                status = self.parse_XML(output, run.exit_code.value, run.was_timeout)
```

Comment:
I think this is buggy, right? `_prase_XML` is expected to be passed an `ProcessExitCode` instance, but gets only the return code here.

Reply:
fixed in a6dbfcbc3fefc99d6a7a99e9a4040b7dee88e614